recipe/recipe.yaml

```diff
@@ -0,0 +1,81 @@
+context:
+  name: "networkx"
+  version: 3.4.2
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/n/${{ name }}/${{ name }}-${{ version }}.tar.gz
```

Comment:
```suggestion
schema_version: 1

context:
  version: 3.4.2

package:
  name: networkx
  version: ${{ version }}

source:
  url: https://pypi.io/packages/source/n/networkx/networkx-${{ version }}.tar.gz
```

Reply:
Thanks for the review @Schefflera-Arboricola !

`schema_version` is optional and is 1 by default https://github.com/prefix-dev/recipe-format/blob/be58bcc167cb743813a32d258fb70639611474f9/schema.json#L3563

The other changes are just aesthetic changes :)

Reply:
I used `conda-recipe-manager` too locally, but it is supposed to be a suggestion and we can iterate on the converted output.